README.md

```diff
@@ -1,114 +1,51 @@
 [![Build Status](https://travis-ci.org/prebid/prebid-server.svg?branch=master)](https://travis-ci.org/prebid/prebid-server)
 
-# prebid-server
-Server side component to offload prebid processing to the cloud
-
-# Discussion group for development / adapter testing
-http://redditadops.slack.com channel headerbidding-dev
-
-# Current Status of Adapters (Sep 20, 2017)
```

Comment:
I understand removing this from the README, but am wondering if we're going to track this anywhere else?

I think it might make sense to have documentation somewhere of the current status of adapters. I'm thinking particularly about which adapters are live for mobile vs web and what formats they support.

Reply:
yes... that's a great idea. Could you log a feature request issue for it?

I was hoping to track things like this through APIs. So for example, right now `/bidders/params` returns json-schemas for the bidder params that everyone expects.

I was expecting to need to add a `GET /bidders` endpoint which returns something simpler, like:

```
[
  "appnexus",
  "rubicon",
  ...
]
```

I could imagine a `/bidders/media-types` which returns something like:

```
{
  "appnexus": ["banner", "video"],
  "index": ["banner"]
}
```

Basic motivation is: docs fall out of date all the time. Since we're running a server anyway, we should be able to generate these things from the code, so they're guaranteed to be up to date.

It'll also help with the fact that different companies might deploy different versions of PBS... so the features might be different depending on whose deploy you're using.

Reply:
Yup that makes a lot of sense. Sounds great, will log a feature request with this info.